models/migrations/v226.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"xorm.io/xorm"
+)
+
+func renameProjectBoardsToColumns(x *xorm.Engine) error {
+	sess := x.NewSession()
+	defer sess.Close()
+	if err := sess.Begin(); err != nil {
+		return err
+	}
+
+	if _, err := sess.Exec("ALTER TABLE `project_board` RENAME TO `project_column`;"); err != nil {
+		return err
+	}
+
+	if _, err := sess.Exec("ALTER TABLE `project_issue` RENAME COLUMN `project_board_id` TO `project_column_id`;"); err != nil {
```

Comment:
in migrations.go there is a helper to rename columns because in some dbs (sqlite) renames don't actually work as expected

Reply:
Do you mean `recreateTable`? I am not sure how to use it, so it copies over all data from the old to the new column name.

Reply:
Seems like there was no helper for renaming yet, so I added two tiny ones.